opencti-platform/opencti-graphql/src/types/event.d.ts

```diff
@@ -52,6 +53,7 @@ interface UpdateEvent extends StreamDataEvent {
   context: {
     patch: Array<Operation>;
     reverse_patch: Array<Operation>;
+    relatedRestrictions?: { markings: string[] };
```

Comment:
Can you use camel case here ? related_restrictions 
Thanks

Reply:
off course. by bad